### PR TITLE
Exclude the omnipresent locations from code coverage

### PR DESCRIPTION
**Pull request by m-Peter**
Work towards: https://github.com/onflow/developer-grants/issues/132

## Description

Certain smart contracts, which are essential for the functioning of the Flow blockchain, are not needed in the context of coverage reporting. We exclude them, in order to avoid skewing the coverage report metrics.

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence-lint/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Review by SupunS — COMMENTED** [review]
Nice idea!

File: test/emulator_backend.go [hunk 1]
```diff
@@ -79,8 +81,26 @@ func NewEmulatorBackend(
...
+			"FlowStorageFees":    chain.ServiceAddress().HexWithPrefix(),
+		}
+		for name, address := range contracts {
+			addr, _ := common.HexToAddress(address)
+			location := common.AddressLocation{
+				Address: addr,
+				Name:    name,
+			}
+			coverageReport.ExcludeLocation(location)
+		}
```
[the remark this answers is not in the document]
Comment (m-Peter) [reply test/emulator_backend.go:103]: That's a good idea :+1: 
However, for all of them, I need the chain:
```go
chain := blockchain.GetChain()
```
which I obtain from the `blockchain` object. Can I get it somehow else, so I can move the initialization to a global variable?

[the remark this answers is not in the document]
Comment (SupunS) [reply test/emulator_backend.go:103]: I see. That makes sense. I didn't notice that the first time 👍 

I'm not sure if we can get that without the chain. Maybe it's also fine to keep it this way, given that people might only create one (or a handful number of) blockchain(s) during tests, most probably.

**Review by SupunS — APPROVED** [review]

File: test/emulator_backend.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-Peter) [reply test/emulator_backend.go]: After some digging on the `flow-go` repository, I found a way to make this possible. Updated in https://github.com/onflow/cadence-tools/pull/110/commits/37e5ef4b5f0e36a905273dc2bcb9e1e3b64a65f7.

**Review by SupunS — APPROVED** [review]
Nice!

**Review by turbolent — APPROVED** [review]
👍

**Pull request merged**